src/cd-mapping-db.c

```diff
@@ -67,7 +67,7 @@ cd_mapping_db_open (CdMappingDb *mdb,
 		    GError  **error)
 {
 	CdMappingDbPrivate *priv = GET_PRIVATE (mdb);
-	g_autofree gchar *error_msg = NULL;
+	gchar *error_msg = NULL;
```

Comment:
I think it should be `char` not `gchar` if not allocated by Glib.

Reply:
(as in, this pattern is widely used when searching by `sqlite3_free` and they all use `gchar *`)

Reply:
I think it's a useful distinction tbh.

Reply:
sure, i could add another commit to change all the existing uses of this if you want

Reply:
Yes please! Thanks.

Reply:
done. there's also `const gchar *statement` which is usually set purely with `= "";` and not allocated via glib apis either. those too?